pkg/preference/preference.go

```diff
@@ -141,6 +151,9 @@ type OdoSettings struct {
 
 	// RegistryList for telling odo to connect to all the registries in the registry list
 	RegistryList *[]Registry `yaml:"RegistryList,omitempty"`
+
+	// RegistryCacheTime how long odo should cache information from registry
+	RegistryCacheTime *int `yaml:"RegistryCacheTime,omitempty"`
```

Comment:
we need to also update the help message
```
# Set a preference value in the global preference
  odo preference set UpdateNotification false
  odo preference set NamePrefix "app"
  odo preference set Timeout 20
  odo preference set BuildTimeout 40
  odo preference set PushTimeout 30
  odo preference set Experimental true
  odo preference set PushTarget docker
```

Reply:
this is just an example, I don't think that we need to show example for all preference.

If you scroll up to `Available Parameters` section you will see description for `RegistryCacheTime` option

Reply:
Sorry didn't notice the `Available Parameters` section.

Reply:
> Sorry didn't notice the `Available Parameters` section.

I had the same problem first time :-) This is why I was thinking about reducing the number of examples that we have there :-)